react-main/src/app/screens/Game/components/Move/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component, Fragment } from 'react';
+
+class Move extends Component {
+  handleJumpTo = move => {
+    this.props.jumpTo(move);
+  };
+
+  render() {
+    const { history, jumpTo, winner, xIsNext } = this.props;
+
+    let status;
```

Comment:
I would define ``const status = winner ? `The winner is: ${winner}` : `Next player is: ${xIsNext ? 'X' : 'O'}`;``